### ISSUE TITLE
styled-jsx-plugin-postcss not work with node12

### ISSUE THREAD

**Issue by superkoh** [giuseppeg/styled-jsx-plugin-postcss#34]
sample project: https://github.com/superkoh/styled-jsx-plugin-postcss-not-work-with-node12

when use node 12, the process will hang at

```
processor(cssWithPlaceholders, settings)
    .then(resolved)
    .catch(resolved)
loopWhile(() => wait)
```

seems promise can not be returned.

How to reproduce:
1. clone the project: https://github.com/superkoh/styled-jsx-plugin-postcss-not-work-with-node12
2. npm install
3. npm run dev

when use node 10, all seems ok. when use node 12, hang at the code above

**Comment by giuseppeg** [comment]
should be related to https://github.com/giuseppeg/styled-jsx-plugin-postcss/issues/31

**Comment by giuseppeg** [comment]
should be fixed in #35
